libp2p/NodeTable.cpp

```diff
@@ -199,7 +199,17 @@ void NodeTable::doDiscover(NodeId _node, unsigned _round, shared_ptr<set<shared_
 	m_timers.schedule(c_reqTimeout.count() * 2, [this, _node, _round, _tried](boost::system::error_code const& _ec)
 	{
 		if (_ec)
-			clog(NodeTableWarn) << "Discovery timer canceled!";
+			clog(NodeTableWarn) << "Discovery timer canceled: " << _ec.value() << _ec.message();
```

Comment:
0 level of required verbosity (`NodeTableWarn::verbosity`) for this message is too low.

all developer-level notices should be 2 or greater. for low-level stuff like this 4 or greater.